### PR TITLE
Add init containers for deployment and statefulset.

### PR DESCRIPTION
**Pull request by MemberIT**

### CODE REVIEW

File: charts/victoria-metrics-agent/values.yaml [hunk 1]
```diff
@@ -129,6 +129,11 @@ extraContainers: []
 #   - name: config-reloader
 #     image: reloader-image
 
+initContainers: 
+  {}
```
Comment (zekker6) [inline charts/victoria-metrics-agent/values.yaml:133]:
```suggestion
  []
```

As far as I can see, the default value here should be an empty list?

**Review by zekker6 — COMMENTED** [review]
Would you mind also updating [CHANGELOG](https://github.com/VictoriaMetrics/helm-charts/blob/master/charts/victoria-metrics-agent/CHANGELOG.md) with description of this change? (similar to https://github.com/VictoriaMetrics/helm-charts/pull/808/commits/e59f034fdb1ede6174a9e6b2edfd769c6bea8eee or other changelog updates)

**Comment by MemberIT** [comment]
> Would you mind also updating [CHANGELOG](https://github.com/VictoriaMetrics/helm-charts/blob/master/charts/victoria-metrics-agent/CHANGELOG.md) with description of this change? (similar to https://github.com/VictoriaMetrics/helm-charts/commit/e59f034fdb1ede6174a9e6b2edfd769c6bea8eee or other changelog updates)

Done

**Review by zekker6 — APPROVED** [review]
LGTM, @MemberIT thanks for adding `initContainers` support!

**Pull request merged**